### ISSUE TITLE
implement genetic algorithm

### ISSUE THREAD

**Issue by Lancasterwu** [GatorEducator/gatorgrouper#238]
# Description of the Issue

## Add Detailed Comment with More Information

implement the genetic algorithm grouping method.
Here is the introduction of this algorithm on [Wikipedia](https://en.wikipedia.org/wiki/Genetic_algorithm)

## Steps to Reproduce Issue

Since this is a new feature, so there is no steps to reproduce this issue.
However, this file should be in `gatorgrouper/utils/` with test case in 'tests/'

### Assigned Developers

@ilikerustoo @Lancasterwu

**Comment by aubreypc** [comment]
It looks like there is a genetic algorithm for grouping in @yeeunmariakim's fork: https://github.com/yeeunmariakim/gatorgrouper/blob/master/genetic_algorithm.py
This might be a good starting point for this feature?

**Comment by Lancasterwu** [comment]
Yes, I have emailed her and received reply that we can start based on that file. However, we still need to follow MIT license.

**Comment by enpuyou** [comment]
Hey, you guys might want to check out this link. http://www.otletinstitute.org/wikics/Grouping_Genetic_Algorithms.html. It has a description of grouping genetic algorithm(GGA) that is an example of applying GA on grouping problems. I feel this will suit our project better
